RFC-676-Contributor-Council-Charter.md

```diff
@@ -0,0 +1,221 @@
+project. Therefore, AWS is committed to creating more robust and clearly-defined channels for soliciting community feedback, and acting on that
+feedback. We are committed to increasing opportunities and building new processes for accepting more community contributions to the codebase.
+3. **Provide visibility into AWS-led initiatives.** We will provide the CDK’s community with visibility into our roadmap and AWS-led contributions.
+We will give the community a voice and a hand in our decision-making processes. We must communicate our roadmaps, plans, and decisions openly and
+promptly, and deliver on them consistently, for the long-term success of CDK.
+
+## Decision-making by the Council
+
+The Council will meet monthly to start, but may increase or decrease meeting frequency at its discretion. Similarly, the length of the meetings will
+be determined by the Council and can be increased or decreased at the Council’s discretion. We anticipate Council members will spend less than 5
+hours a month on Council activities.
+
+Portions of the sessions will be recorded and published to YouTube. Discussion topics under a Non-Disclosure Agreement (NDA) will be redacted from
+the recording before it is published.
+
+### The Council meetings include two sections
+
+(1) Town hall
+
+* The AWS CDK team will report on project release priorities and the status of AWS-led features in each Council meeting. Council members will have
+an opportunity to ask questions and deliver feedback.
+* Community representatives will report on news and share updates on community-led features. Council members will have an opportunity to ask
+questions and deliver feedback.
+
+(2) New proposals
+
+* The Council will review and debate RFCs/features, feature docs, or new processes brought forth by the CDK community and AWS. AWS may not be able
+to report on all AWS-led projects when we are bound by legal requirements, customer agreements, or when it would put AWS at a competitive
+disadvantage to report on some project.
+* Anyone in the CDK community may propose topics for discussion by the Council. The agenda is finalized one week before each meeting to allow time
+for pre-reading and preparation. Proposers are invited to the meeting to present and advocate for their change. If they are unable to attend,
+their attendance is not required for the Council to make a decision.
+* The Council discusses the change’s appropriateness, merits, scope of work, and whether the change can be community-led or if it will be built by
+internal Amazon resources.
+* Decisions are completed and recorded through the following mechanism:
+  * Each Council member enters an opinion on whether the feature is viable including a short explanation. The opinions and reasons behind them are
+recorded on the change proposal. Opinion submissions follow the [Chatham House Rule](https://www.ibabs.com/en/glossary/chatham-house-rule/#:~:text=The%20Chatham%20House%20Rule%20reads,')
```

Comment:
Chatham House has as many bad reasons to be avoided as good reasons to be used.

Reply:
Can you please give examples?